<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HistogramDialog</class>
 <widget class="QDialog" name="HistogramDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>808</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>510</y>
     <width>181</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QtCharts::QChartView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>661</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>430</y>
     <width>611</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>470</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QComboBox" name="mode">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>96</width>
     <height>35</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Red</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Green</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Blue</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Average</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="buttonOtsu">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>470</y>
     <width>109</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>Otsu</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtCharts::QChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">QChartView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>HistogramDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>338</x>
     <y>510</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>HistogramDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>406</x>
     <y>510</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>691</x>
     <y>445</y>
    </hint>
    <hint type="destinationlabel">
     <x>720</x>
     <y>489</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
